config/config.go

```diff
@@ -395,24 +440,83 @@ func (s SATranslationConfig) ToMaps(inBound bool) (map[string]map[string]string,
 	return reqMap, respMap
 }
 
-// AsLocalToRemoteBiMaps converts the flat list of namespace + local/remote pairs into a map of BiMaps, with local->remote
+type SearchAttributeTranslation struct {
+	inner    map[string]collect.StaticBiMap[string, string]
+	inverted bool
+}
+
+func (s SearchAttributeTranslation) Inverse() SearchAttributeTranslation {
+	return SearchAttributeTranslation{
+		inner:    s.inner,
+		inverted: !s.inverted,
+	}
+}
+func (s SearchAttributeTranslation) withInvert(namespace string) (collect.StaticBiMap[string, string], bool) {
+	m, found := s.inner[namespace]
+	if !found {
+		return nil, false
+	}
+	if s.inverted {
+		return m.Inverse(), true
+	}
+	return m, true
+}
+func (s SearchAttributeTranslation) Get(namespace string, searchAttr string) string {
+	if m, ok := s.withInvert(namespace); ok {
+		return m.Get(searchAttr)
+	}
+	return ""
+}
+func (s SearchAttributeTranslation) GetExists(namespace string, searchAttr string) (string, bool) {
+	if m, ok := s.withInvert(namespace); ok {
+		return m.GetExists(searchAttr)
+	}
+	return "", false
+}
+func (s SearchAttributeTranslation) LenNamespaces() int {
+	return len(s.inner)
+}
+func (s SearchAttributeTranslation) Len(namespace string) int {
```

Comment:
not used? i would probably have a method to return per namespace map something `GetMap(namespace)` and then call `Len`